### DISCUSSION TITLE
Are the dependencies graphs images generated by any tool?

### DISCUSSION THREAD

**Discussion by henry1988** [android/nowinandroid#446] [Q&A] [answered]
Hi, I notice that every module has an image with its dependencies graph. I was curious if an automated tool/plugin generates this or if it is created manually. 

Thanks for sharing this project.

**Comment by jisungbin** [comment, marked as answer]
It was created automatically.
See #310.

**Reply by henry1988** [reply]
Thanks @jisungbin , hope they release the plugin as open source soon.
